mne/stats/regression.py

```diff
@@ -134,3 +142,197 @@ def _fit_lm(data, design_matrix, names):
         mlog10_p_val[predictor] = -np.log10(p_val[predictor])
 
     return beta, stderr, t_val, p_val, mlog10_p_val
+
```

Comment:
This *is* the part of epoch regression that's later returned as a namedtupled :) raw regression only outputs betas.